lib/train/transports/winrm_connection.rb

```diff
@@ -183,7 +183,9 @@ def session(retry_options = {})
     #
```

Comment:
@crepetl Could you please add `# rubocop:disable Metrics/ClassLength` to line 30 https://github.com/chef/train/pull/171/files#diff-f2edeefbb087259b847aa6f5ded7d0f4L30 similar to https://github.com/chef/train/blob/master/lib/train/transports/ssh_connection.rb#L32 This would make rubocop happy.